src/react/PageRoute.jsx

```diff
@@ -22,7 +22,7 @@ export default function PageRoute(props) {
     if (match) {
       sendPageEvent();
     }
-  }, [JSON.stringify(match)]);
```

Comment:
This change makes me nervous - are we sure that the `match` object changes with the same frequency as the stringified version of it?  I'd be worried that we end up sending page events far more frequently, which could be pretty disruptive to our analytics.  If this was just to satisfy the linter I'd probably revert this and leave it as-is.  We should try not to make code changes when just updating the build library.